index.Rmd

```diff
@@ -247,7 +247,7 @@ test2 = test1 %>% mutate(plot = map2(data, County,
                                      labs(caption="Solid lines indicate pathogen prevalence from sample specimens in County")+
                                      ggtitle(glue("{.y}"))))
 ```
-
+ 
```

Comment:
All the code before this is very impressive, but I didn't really know what was happening. If you could provide a brief overview every so often to explain what you're doing, that would be very helpful.